patches/server/0188-Call-PaperServerListPingEvent-for-legacy-pings.patch

```diff
@@ -69,12 +70,12 @@ index 0000000000000000000000000000000000000000..74c012fd40491f1d870fbc1aa8c318a2
 +    }
 +
 +    public static String getMotd(PaperServerListPingEvent event) {
-+        return getFirstLine(event.getMotd());
++        return getFirstLine(PlainTextComponentSerializer.plainText().serialize(event.motd()));
```

Comment:
This looks like an api break?

Reply:
nah, you right. I was just replacing uses of deprecated stuff, but this is a valid use of getMotd() so I'll just suppress that use.